custom/icds/templates/icds/messaging/indicators/kan/dpt3_and_measles_due.txt

```diff
@@ -0,0 +1 @@
+{{ child_name }} is due for DPT3 and Measles for this month and have not received the vaccination
```

Comment:
And another.

Reply:
@millerdev This was left out intentionally because this message is not active. The Ministry removed this message from the scope of release so we don't intend to translate this message at this time.